### PR TITLE
readd OK providers

### LINKED ISSUE #76: re-add OK providers with the new data structure
It should be a no-brainer to re-add the providers with `status: OK` from the old directory.

### PR DESCRIPTION
**Pull request by missytake**
closes #76 

maaaaybe it would have been faster manually. But now I learned how to parse yml & xml :)

What's isn't copied is the markdown part - but for the OK providers we didn't want to include it anyway, right?

### CODE REVIEW

**Comment by pabzm** [comment]
It's cool that you learned how to handle YAML and XML! But in this repo we need the results of running your script, not the script itself, don't we?

**Comment by missytake** [comment]
I didn't look yet which default values we can skip because they are tried automatically anyway...

File: _providers/all-inkl.com.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+domains: (custom)
```
Comment (pabzm) [inline _providers/all-inkl.com.md]: Please don't include providers for custom domains.

File: _providers/bluewin.ch.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+domains: bluewin.ch
```
Comment (pabzm) [inline _providers/bluewin.ch.md]: `domains` must be an array, this is a string.

File: _providers/hosteurope.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+domains: (custom)
```
Comment (pabzm) [inline _providers/hosteurope.md]: No providers for custom domains, please.

File: _providers/kontent.com.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+---
+domains:
+- kontent.com
+- (custom)
```
Comment (pabzm) [inline _providers/kontent.com.md]: No custom domains, please.

File: _providers/mailbox.org.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+---
+domains:
+- mailbox.org
+- secure.mailbox.org
+- (custom)
```
Comment (pabzm) [inline _providers/mailbox.org.md]: No custom domains, please.

File: _providers/manitu.de.md [hunk 6]
```diff
@@ -0,0 +1,7 @@
+---
+domains: manitu.de
+last_checked: 2019-04
+name: Manitu (Custom Domain)
```
Comment (pabzm) [inline _providers/manitu.de.md]: Is this for custom domains or for manitu.de?

File: _providers/strato.de.md [hunk 7]
```diff
@@ -0,0 +1,7 @@
+---
+domains: (custom)
```
Comment (pabzm) [inline _providers/strato.de.md]: No providers for custom domains, please.

File: _providers/zoho.com.md [hunk 8]
```diff
@@ -0,0 +1,9 @@
+---
+domains:
+- zoho.com
+- (custom)
```
Comment (pabzm) [inline _providers/zoho.com.md]: No custom domains, please

File: _providers/manitu.de.md (hunk 6 above)
Comment (missytake) [reply _providers/manitu.de.md]: There are of course some manitu.de addresses, e.g. support@manitu.de, but they don't really seem to be a mail provider. I took them out.

**Comment by missytake** [comment]
I had a bug in the python script, which left out the server: information from the TB autoconfig database. I went back in time, fixed it, printed the output, and copy-pasted it manually. Now it should be complete.

This can probably be merged, except that there is still the discussion in #76 regarding whether we want to keep them at all.

File: _providers/verizon.md [hunk 9]
```diff
@@ -0,0 +1,12 @@
+---
+name: Verizon
+status: OK
+domains: 
+  - verizon.net
+server:
+  - hostname: smtp.verizon.net
```
Comment (r10s) [inline _providers/verizon.md]: please specify either smtp+imap or nothing.

if in doubt, just delete the provider

File: _providers/old/tiscali.it.md [hunk 10]
```diff
@@ -1,9 +1,7 @@
...
+last_checked: 2019-12
 name: Tiscali
+status: OK
 website: https://tiscali.it
-domains: tiscali.it
-credentials: emailPass
-status:
- state: OK
- date: 2019-12
 ---
```
Comment (pabzm) [inline _providers/old/tiscali.it.md]: Why did you change the file in `old/` here? I guess this was meant to be moved to the parent directory?

Comment (r10s) [reply _providers/old/tiscali.it.md]: i think it can just be deleted, it was already updated in #72

Comment (pabzm) [reply _providers/old/tiscali.it.md]: Oh, I missed that. Thanks!
